docs/v4.0-draft1/resources-and-data-models/aisp/standing-orders.md

```diff
@@ -104,12 +104,21 @@ If the ReadPAN permission is granted by the PSU, the ASPSP may choose to populat
-| Frequency |1..1 |OBReadStandingOrder6/Data/StandingOrder/Frequency |Individual Definitions: <br/>NotKnown - Not known <br/>EvryDay - Every day<br/> EvryWorkgDay - Every working day<br/> IntrvlDay - An interval specified in number of calendar days (02 to 31)<br/> IntrvlWkDay - An interval specified in weeks (01 to 09), and the day within the week (01 to 07)<br/> WkInMnthDay - A monthly interval, specifying the week of the month (01 to 05) and day within the week (01 to 07)<br/> IntrvlMnthDay - An interval specified in months (between 01 to 06, 12, 24), specifying the day within the month (-05 to -01, 01 to 31)<br/> QtrDay - Quarterly (either ENGLISH, SCOTTISH, or RECEIVED)<br/> ENGLISH = Paid on the 25th March, 24th June, 29th September and 25th December.<br/> SCOTTISH = Paid on the 2nd February, 15th May, 1st August and 11th November.<br/> RECEIVED = Paid on the 20th March, 19th June, 24th September and 20th December.  <br/><br/><br/>Individual Patterns: <br/>NotKnown (ScheduleCode)<br/>EvryDay (ScheduleCode)<br/> EvryWorkgDay (ScheduleCode)<br/> IntrvlDay:NoOfDay (ScheduleCode + NoOfDay)<br/> IntrvlWkDay:IntervalInWeeks:DayInWeek (ScheduleCode + IntervalInWeeks + DayInWeek)<br/> WkInMnthDay:WeekInMonth:DayInWeek (ScheduleCode + WeekInMonth + DayInWeek)<br/> IntrvlMnthDay:IntervalInMonths:DayInMonth (ScheduleCode + IntervalInMonths + DayInMonth)<br/> QtrDay: + either (ENGLISH, SCOTTISH or RECEIVED) ScheduleCode + QuarterDay<br/><br/> The regular expression for this element combines five smaller versions for each permitted pattern. To aid legibility - the components are presented individually here:<br/>NotKnown <br/>EvryDay<br/> EvryWorkgDay<br/> IntrvlDay:`((0[2-9])\|([1-2][0-9])\|3[0-1])`<br/> IntrvlWkDay:`0[1-9]:0[1-7]`<br/> WkInMnthDay:`0[1-5]:0[1-7]`<br/> IntrvlMnthDay:`(0[1-6]\|12\|24):(-0[1-5]\|0[1-9]\|[12][0-9]\|3[01])`<br/> QtrDay:`(ENGLISH\|SCOTTISH\|RECEIVED)`<br/><br/><br/> Full Regular Expression:<br/> `^(NotKnown)$\|^(EvryDay)$\|^(EvryWorkgDay)$\|^(IntrvlDay:((0[2-9])\|([1-2][0-9])\|3[0-1]))$\|^(IntrvlWkDay:0[1-9]:0[1-7])$\|^(WkInMnthDay:0[1-5]:0[1-7])$\|^(IntrvlMnthDay:(0[1-6]\|12\|24):(-0[1-5]\|0[1-9]\|[12][0-9]\|3[01]))$\|^(QtrDay:(ENGLISH\|SCOTTISH\|RECEIVED))$` |Max35Text | | `^(NotKnown)$\|^(EvryDay)$\|^(EvryWorkgDay)$\|^(IntrvlWkDay:0[1-9]:0[1-7])$\|^(WkInMnthDay:0[1-5]:0[1-7])$\|^(IntrvlMnthDay:(0[1-6]\|12\|24):(-0[1-5]\|0[1-9]\|[12][0-9]\|3[01]))$\|^(QtrDay:(ENGLISH\|SCOTTISH\|RECEIVED))$` |
+| MandateRelatedInformation | 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation ||OBMandateRelatedInformation | | |
+| MandateIdentification | 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/MandateIdentification ||TODO | | |
+| Classification | 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/OBClassification1Code |FIXE<br>USGB<br>VARI|OBClassification1Code | | |
```

Comment:
Path for Classification should be OBReadDirectDebit2/Data/DirectDebit/MandateRelatedInformation/Classification